File: JavaDemo/app/src/main/java/com/example/javademo/PostApi.java
package com.example.javademo;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface PostApi {
    String BASE_URL = "https://jsonplaceholder.typicode.com/";

    @GET("posts")
    Call<List<Post>> getPosts();
}

File: KotlinDemo/app/src/main/java/com/example/kotlindemo/PostApi.kt
package com.example.kotlindemo

import retrofit2.Call
import retrofit2.http.GET

interface PostApi {
    object Const {
        val BASE_URL: String
            get() = "https://jsonplaceholder.typicode.com/"
    }

    @GET("posts")
    fun getPosts(): Call<List<Post>>
}